tests/test_cliCmdTestIngest.py

```diff
@@ -55,15 +59,16 @@ def run_test(self, inputs, expectedKwargs):
             The expected arguments to the ingestRaws command function, keys are
             the argument name and values are the argument value.
         """
-        runner = click.testing.CliRunner(env=DAF_BUTLER_MOCK)
+        runner = click.testing.CliRunner()
         result = runner.invoke(butler.cli, inputs)
         self.assertEqual(result.exit_code, 0, f"output: {result.output} exception: {result.exception}")
-        verifyFunctionInfo(self, result.output, "ingestRaws", (), expectedKwargs)
+        lsst.obs.base.cli.test.ingestRaws.mock.assert_called_with(**expectedKwargs)
 
     def test_repoAndOutput(self):
         """Test the most basic required arguments, repo and output run"""
         expected = makeExpectedKwargs(repo="here", output_run="out")
-        self.run_test(["ingest-raws", "here", "--output-run", "out"], expected)
+        self.run_test(["ingest-raws", "here",
```

Comment:
Is this change needed?

Reply:
it's not necessary, but makes this test function look like the rest of the test functions in the suite, making it so that in the tuple of parameters each option+value is on a separate line (the first line is consistently a special case, subcommand+repo), which IMO is easier to read.